### ISSUE TITLE
Adjust 'httpd' service with systemd / puppet syntax

### ISSUE THREAD

**Issue by jeff1evesque** [jeff1evesque/drupal-demonstration#264]
We will need to start, and restart the httpd service, using the systemd / puppet syntax, within `default.pp`, and `ssl.pp`.

**Comment by jeff1evesque** [comment]
24d3685: there is already the same puppet stanza `restart-httpd` within `default.pp`.

**Comment by jeff1evesque** [comment]
We are currently receiving the following error traceback:

```bash
...
==> default: Running provisioner: puppet...
==> default: Running Puppet with environment development...
==> default: Notice: Compiled catalog for drupal-demonstration.com in environmen
t development in 0.99 seconds
==> default: Notice: /Stage[main]/Main/Package[mod_ssl]/ensure: created
==> default: Notice: /Stage[main]/Main/File[/etc/httpd/ssl/]/ensure: created
==> default: Notice: /Stage[main]/Main/Exec[create-ssl]: Triggered 'refresh' fro
m 1 events
==> default: Notice: /Stage[main]/Main/Exec[adjust-firewalld-ssl]: Triggered 're
fresh' from 1 events
==> default: Notice: /Stage[main]/Main/Exec[restart-firewalld-ssl]: Triggered 'r
efresh' from 1 events
==> default: Notice: /Stage[main]/Main/Exec[assign-ssl-certificate]: Triggered '
refresh' from 1 events
==> default: Notice: /Stage[main]/Main/Exec[assign-ssl-key]: Triggered 'refresh'
 from 1 events
==> default: Notice: /Stage[main]/Main/Exec[restart-httpd-2]/returns: Job for ht
tpd.service failed because the control process exited with error code. See "syst
emctl status httpd.service" and "journalctl -xe" for details.
==> default: Error: /Stage[main]/Main/Exec[restart-httpd-2]: Failed to call refr
esh: systemctl restart httpd returned 1 instead of one of [0]
==> default: Error: /Stage[main]/Main/Exec[restart-httpd-2]: systemctl restart h
ttpd returned 1 instead of one of [0]
==> default: Notice: Applied catalog in 10.25 seconds
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

Therefore, we will temporarily replace `ssl.pp`, with just the following, before `vagrant destroy`, followed by `vagrant up`:

```bash
...
## restart apache server
exec {'restart-httpd-2':
    command => 'systemctl restart httpd',
    refreshonly => true,
}
...
```

**Comment by jeff1evesque** [comment]
The above immediate comment was resolved via #266.
